go_covert_lib/client/src/screens/ConfigScreen.jsx

```diff
@@ -0,0 +1,307 @@
+                        [key]: {
+                          ...opt,
+                          Value: e.target.value,
+                        },
+                      },
+                    },
+                  },
+                  ...processors.slice(i + 1, processors.length + 1),
+                ]),
+              };
+              switch (opt.Type) {
+                case 'ipv4':
+                  return (
+                    <IPInput {...propsForComponent} />
+                  );
+                case 'i8':
+                case 'u16':
+                case 'u64':
+                case 'exactu64':
+                  if (opt.Range) {
+                    propsForComponent = {
+                      ...propsForComponent,
+                      min: opt.Range[0],
+                      max: opt.Range[1],
+                    };
+                  }
+                  return (
+                    <NumberInput
+                      {...propsForComponent}
+                      parentOnChange={e => setProcessors([
+                        ...processors.slice(0, i),
+                        {
+                          ...processor,
+                          Data: {
+                            ...processor.Data,
+                            [processor.Type]: {
+                              ...processor.Data[processor.Type],
+                              [key]: {
+                                ...opt,
+                                Value: parseInt(e.target.value) || 0,
+                              },
+                            },
+                          },
+                        },
+                        ...processors.slice(i + 1, processors.length + 1),
+                      ])}
+                    />
+                  );
+                case 'bool':
+                  return (
+                    <Checkbox
+                      {...propsForComponent}
+                      parentOnChange={e => setProcessors([
+                        ...processors.slice(0, i),
+                        {
+                          ...processor,
+                          Data: {
+                            ...processor.Data,
+                            [processor.Type]: {
+                              ...processor.Data[processor.Type],
+                              [key]: {
+                                ...opt,
+                                Value: e.target.checked,
+                              },
+                            },
+                          },
+                        },
+                        ...processors.slice(i + 1, processors.length + 1),
+                      ])}
+                    />
+                  );
+                case 'select':
+                  return (
+                    <Select
+                      {...propsForComponent}
+                      items={opt.Range}
+                    />
+                  );
+                default:
```

Comment:
There are several of these parameters that are unimplemented at this time. Those will have to be updated in the future.